x-pack/plugins/monitoring/public/alerts/legacy_alert/legacy_alert.tsx

```diff
@@ -18,6 +18,9 @@ export function createLegacyAlertTypes(): AlertTypeModel[] {
       name: LEGACY_ALERT_DETAILS[legacyAlert].label,
       description: LEGACY_ALERT_DETAILS[legacyAlert].description,
       iconClass: 'bell',
+      documentationUrl(docLinks) {
+        return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/kibana/${docLinks.DOC_LINK_VERSION}/kibana-alerts.html`;
```

Comment:
For these, I'd link to https://www.elastic.co/guide/en/kibana/current/cluster-alerts.html as they are technically Kibana alerts, but are really the same watches under the hood that have always existed. We will soon change these alerts to not rely on the watches, and when we do that, we will update this link.

Reply:
Fixed in https://github.com/elastic/kibana/pull/81526/commits/21aac5eb57803e9df9ff83b2ee1f28d132bdcf5e.